tests/automated_tests/test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright (c) 2023 Robert Bosch GmbH
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+exit_if_not_successful() {
+   if [ $? -ne 0 ]
+   then
+      exit $?
+   fi
+}
+
+for runtime in runtime-k3d runtime-kanto runtime-local
+do
+   ./tests/automated_tests/runtime_tests.sh $runtime
+   exit_if_not_successful
+done
+
+pytest tests/automated_tests/requirements_tests.py
+exit_if_not_successful
```

Comment:
why testing this twice?

Reply:
what is tested twice?

Reply:
runtime_tests.sh and requirements_test.sh are already executed during the example_test

Reply:
yes, but not with example app, if you take a look deeper, this is to ensure that example app works within each runtime

Reply:
you don't test at all if the example app is running, you just test the integration tests of the runtime without your app

Reply:
but they are building and starting the app which is in the test-folder of the runtimes, not the one we have in the template